packages/style/2016-12-01/index.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
-  BorderRadius: require('../modules/border-radius'),
-  Hovers: require('../modules/hovers'),
-  Skins: require('../modules/skins'),
-  Spacing: require('../modules/spacing')
-}
\ No newline at end of file
+  BorderRadius: require('../modules/border-radius.js'),
+  Hovers: require('../modules/hovers.js'),
+  Skins: require('../modules/skins.js'),
+  Spacing: require('../modules/spacing.js')
```

Comment:
The module mapper looks for calls to `require` that don't include `js` or `jsx` in the module name. Implicit file extensions get us into trouble here. Odd, huh?